tests/test_query_registry.py

```diff
@@ -0,0 +1,21 @@
+""" Unit tests for query registry
+
+Copyright (c) 2021-, Tellor Development Community
+Distributed under the terms of the MIT License.
+"""
+from telliot.queries.query_registry import query_registry
+from telliot.queries.query_registry import QueryRegistry
+
+
+def test_query_registry():
```

Comment:
Maybe rename to test_query_export_import, since there's a lot more to the actual registry that could be tested (like the legacy queries)?